1-exercises/D-for-loop/exercise.js

```diff
@@ -11,23 +11,21 @@
 */
 
 const WRITERS = [
-    "Virginia Woolf",
-    "Zadie Smith",
-    "Jane Austen",
-    "Bell Hooks",
-    "Yukiko Motoya"
-]
-
-const AGES = [
-    59,
-    40,
-    41,
-    63,
-    49
+  "Virginia Woolf",
+  "Zadie Smith",
+  "Jane Austen",
+  "Bell Hooks",
+  "Yukiko Motoya",
 ];
 
+const AGES = [59, 40, 41, 63, 49];
+
 // TODO - Write for loop code here
 
+for (let i = 0; i < WRITERS.length; i++) {
+  console.log(WRITERS[i], "is", AGES[i]);
```

Comment:
We could use a string literal here, for example, `console.log(`${WRITERS[i] is ${AGES[i]} years old`})`

Reply:
It's not rendering properly here, but I used ``` around the sentece